### PR TITLE
Rename ocitools to oci-runtime-tool

### PR DESCRIPTION
**Pull request by mrunalp**
@liangchenye @wking @Mashimiao PTAL

Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by philips** [comment]
LGTM

File: test_runtime.sh [hunk 1]
```diff
@@ -77,9 +77,9 @@ trap cleanup EXIT
 tar -xf  rootfs.tar.gz -C ${TESTDIR}
 cp runtimetest ${TESTDIR}
 
-ocitools generate --output "${TESTDIR}/config.json" "${TEST_ARGS[@]}" --rootfs '.'
+oci-runtime-tool generate --output "${TESTDIR}/config.json" "${TEST_ARGS[@]}" --rootfs '.'
 
-TESTCMD="${RUNTIME} start $(uuidgen)"
+TESTCMD="${RUNTIME} run $(uuidgen)"
```
Comment (wking) [inline test_runtime.sh]: Drive-by change?  More discussion on this change in #184, but I'd rather not do this as part of this PR.

Comment (mrunalp) [inline test_runtime.sh]: @wking It was unintentional. Reverted. Thanks!

**Comment by wking** [comment]
a774b91 looks good to me :).

**Comment by mrunalp** [comment]
@wking @philips Thanks! I'll merge this in and we can fixup things up later if we find anything amiss.

**Comment by wking** [comment]
On Wed, Sep 14, 2016 at 12:25:21PM -0700, Mrunal Patel wrote:
> I'll merge this in and we can fixup things up later if we find anything amiss. 

Ah, we forgot to bump MAINTAINERS and nominate @liangchenye to put him
back in [1,2].

[1]: https://github.com/opencontainers/tob/pull/18/files#r77909738
[2]: https://github.com/opencontainers/tob/pull/18/files#diff-07f935bfb8e1871445d8bf2ad3759f9bR30
